user/apps.py

```diff
@@ -1,3 +1,5 @@
+"""App configuration.
+"""
```

Comment:
I am not sure if we need a description here since it does not add information to the code we see.
@kawindme What do you think?